skeleton-es2016-aspnetcore/src/skeleton-navigation-es2016-vs/Views/Home/Index.cshtml

```diff
@@ -1,12 +0,0 @@
-﻿<div aurelia-app="main">
-    <div class="splash">
-        <div class="message">Aurelia Navigation Skeleton</div>
-        <i class="fa fa-spinner fa-spin"></i>
-    </div>
-
-    <script src="jspm_packages/system.js"></script>
-    <script src="config.js"></script>
-    <script>
-        System.import('aurelia-bootstrapper');
-    </script>
-</div>
```

Comment:
Not needed AFAIK - I have "app.UseStaticFiles();" in the startup.cs source. You can browse via right click on index.html and "view in browser" or from the command line do "dotnet run" and prowse to localhost:5000\index.html.